### PR TITLE
x11-misc/dxpc: EAPI bump 0 -> 6

### PR DESCRIPTION
**Pull request by Moiman**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

File: x11-misc/dxpc/dxpc-3.9.2-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND="x11-libs/libXt
+	>=dev-libs/lzo-2"
+DEPEND="${RDEPEND}
+	x11-proto/xproto"
+
+DOCS=( CHANGES README TODO )
```
Comment (SoapGentoo) [inline x11-misc/dxpc/dxpc-3.9.2-r2.ebuild]: nuke this, all of these are part of the default set

File: x11-misc/dxpc/dxpc-3.9.2-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+src_install () {
+	emake prefix="${D}/usr" man1dir="${D}/usr/share/man/man1" install
```
Comment (SoapGentoo) [inline x11-misc/dxpc/dxpc-3.9.2-r2.ebuild]: `emake prefix="${ED%/}"/usr man1dir="${ED%/}"/usr/share/man/man1 install`
(always try to make things Gentoo Prefix-safe)

File: x11-misc/dxpc/dxpc-3.9.2-r2.ebuild (hunk 1 above)
Comment (Moiman) [inline x11-misc/dxpc/dxpc-3.9.2-r2.ebuild]: There is also README.minigw which doesn't sound so useful.

Comment (SoapGentoo) [inline x11-misc/dxpc/dxpc-3.9.2-r2.ebuild]: fair enough, keep it

**Pull request closed without merging**